libraries/AP_NavEKF2/AP_NavEKF2.cpp

```diff
@@ -554,6 +558,15 @@ const AP_Param::GroupInfo NavEKF2::var_info[] = {
     // @RebootRequired: True
     AP_GROUPINFO("EXTNAV_DELAY", 50, NavEKF2, _extnavDelay_ms, 10),
 
+    // @Param: FLOW_MASK
+    // @DisplayName: Optical flow use bitmask
+    // @Description: Bitmask controlling if the optical flow dara is fused into the main navigation estimator and/or the terrain estimator.
+    // @User: Advanced
+    // @Values: 0:None,1:Navigation,2:Terrain,3:Both
```

Comment:
Is it "Navigation" or is it perhaps "Position Estimation"?

Reply:
Navigation is a general term which includes the monitoring of movement and is a more accurate description of how the data will be used.  The description in line 563 provides the context with use in the 'main navigation estimator' and will be updated to make is clearer that one is use in the 22-state estimator and the other use in the 1-state estimator which estimates terrain height only.

Reply:
sure ok, sounds good.